docker/dev/celery/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM ubuntu:16.04
+MAINTAINER CloudCV Team
```

Comment:
Please remove `MAINTAINER` line from all dockerfiles since it is deprecated now.